crates/biome_js_analyze/tests/specs/nursery/useNumberProperties/valid.js

```diff
@@ -0,0 +1,12 @@
+Number.parseInt("1");
+
```

Comment:
We should add test for global use of `isFinite` and `isNan` that should not be reported by the rule.

We could also add some example with local variable with the name of Number's properties:
For example, the following code should be valid:

```js
function f(parseFloat) {
  parseFloat("1.1");
}
```

Reply:
Add this. f85acb5